lightning/src/ln/functional_test_utils.rs

```diff
@@ -259,7 +259,17 @@ macro_rules! get_feerate {
 
 macro_rules! get_local_commitment_txn {
 	($node: expr, $channel_id: expr) => {
-		$node.node.channel_state.lock().unwrap().by_id.get_mut(&$channel_id).unwrap().channel_monitor().get_latest_local_commitment_txn()
+		{
```

Comment:
20725f3

I don't get commit message, you should mark than intented goal is to remove Channel.channel_monitor in future PRs, but why referring would_broadcast_at_height ?